repository-hbase/src/test/scala/hbase/util/RowKeyUtilsTest.scala

```diff
@@ -14,23 +16,27 @@ import hbase.model.AdminData
  */
 object RowKeyUtilsTest extends FlatSpec with Matchers {
 
+  private val configuration = Configuration(ConfigFactory.load)
+
   private val TEST_REFERENCE_PERIOD: YearMonth = YearMonth.parse("201707")
   private val TEST_KEY: String = "123456789"
   private val TEST_VAT_ROWKEY = String.join(RowKeyUtils.DELIMITER, "201707", TEST_KEY)
 
+  private val reverseFlag = configuration.underlying.getBoolean("load.format.reverse")
```

Comment:
Add a test to test with reverse=true and reverse=false, don't refer top the config. You simply need a test for normal key creation and one for reverse key creation to check the generated key is as expected